### ISSUE TITLE
This was the change in the button i was talking about

### ISSUE THREAD

**Issue by anjali-2504** [Raghwendra-Dey/productivity_meter#39]
![IMG_20201208_205649.jpg](https://user-images.githubusercontent.com/75211465/101503310-fa78d580-3997-11eb-9490-1eb16ad848cc.jpg)

**Comment by Raghwendra-Dey** [comment]
@anjali-2504 Actually after #37 I don't think we need any more styling for the buttons. Maybe we need it on later stages when other things are added, or maybe when additional buttons would be added.
Now you can consider contributing to #40

**Comment by Raghwendra-Dey** [comment]
And you can always discuss various ideas regarding this project in the gitter channel. Please join it, if not already done so(link in readme).
